test/info.py

```diff
@@ -315,6 +317,7 @@ def VENDOR_TO_FAMILY(x, y) : return (VENDOR_ID[x] <<8) | y
     0x0423,  # MAX78000FTHR
     0x0424,  # MAX32670EVKIT
     0x0425,  # MAX32650FTHR
+    0x0426,  # MAX32690FTHR
```

Comment:
Shouldn't that be `MAX32690EVKIT` for consistency?

Reply:
Correct, thanks, updated.